src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -39,10 +39,10 @@ function processAuthoredGametes(authoredChallenge, drakes, state) {
             : state.gametes;
 }
 
-function processAuthoredDrakes(authoredChallenge, trial, template) {
+function processAuthoredDrakes(authoredChallenge, trial, template, unscrambledTrialNumber) {
   // takes authored list of named drakes ("mother", etc) and returns an
   // array specific for this template
-  const authoredDrakesArray = template.authoredDrakesToDrakeArray(authoredChallenge, trial);
+  const authoredDrakesArray = template.authoredDrakesToDrakeArray(authoredChallenge, trial, unscrambledTrialNumber);
```

Comment:
I introduced the idea of an `unscrambledTrialNumber` into the `authoredDrakesToDrakeArray` function, to expand the flexibility of the authoring doc a little. Now, you can specify a combination of randomized and non-randomized information in the same challenge definition.

Below, you can see the way I used it in this PR: in level 16, I created 3 sets of parent drakes and matching target drakes to match the spec. I want to be able to see those trials in a random order, *but* I also want to ensure that the first trial only allows the mother's alleles to change, the second only allows manipulation of the father, and the third trial manipulates both. So, to get the correct trial information, I use the "scrambled" number to look up the drake information, and the "unscrambled" number to look up which parents can be manipulated.